chia/server/ws_connection.py

```diff
@@ -391,34 +400,135 @@ async def _wait_and_retry(self, msg: Message, queue: asyncio.Queue):
             self.log.debug(f"Exception {e} while waiting to retry sending rate limited message")
             return None
 
+    async def _potentially_compress(self, message: Message) -> Message:
```

Comment:
this function should probably not be `async`

Reply:
I agree the async can go.